### ISSUE TITLE
having issues for installing foor installing waydroid

### ISSUE THREAD

**Issue by Midnightmadwalk-coder** [Aman9das/Waydroid_Setup_Guide#10]
hi, i dont understand what you meant by temp folder stuff and which archives. could you  please kindly elaborate? @Aman9das
![Screenshot from 2022-08-15 08-01-17](https://user-images.githubusercontent.com/78001660/184568152-7975f7ed-8f45-4219-b56d-f9efd05c7b38.png)

**Comment by Aman9das** [comment]
From step 2 you should have downloaded a `.zip` file. extract the 2 `.img` files inside it into your home directory > temp_folder (for example `/home/aman/temp_folder` for me)

**Comment by Midnightmadwalk-coder** [comment]
thanka. finally installed this. but how can i install playstore in this? could youbplease help?

**Comment by Midnightmadwalk-coder** [comment]
Also idk how to use internet or wifi in waydroid tho. Anything to do about that?

**Comment by Aman9das** [comment]
For playstore the best thing is downloading aurora store

You should be able to use the browser pre-installed in the system.. Unless there is some issue - I'd suggest joining the telegram group then..

**Comment by Midnightmadwalk-coder** [comment]
@Aman9das did you mention the waydroid telegram group or something else? Btw could you explain this step? After adding the 2nd step command for usually waydroid section, how to save it apply the text you've provided which includes (protocol). Reference given in screenshot below
![Screenshot_20220816-101417_Firefox](https://user-images.githubusercontent.com/78001660/184799859-81ef483a-186f-47d3-bda1-ae72ddd908cd.png)

**Comment by Aman9das** [comment]
Yeah the waydroid telegram group.. and save it using "write out" `ctrl+O`

**Comment by Midnightmadwalk-coder** [comment]
hi  , @Aman9das i'd uninstall try to reinstall the waydroid. from your steps in pre-requisite in 2nd step, i'm getting n error like the directgrry is not empty. is this normal?
![Screenshot from 2022-08-17 07-11-22](https://user-images.githubusercontent.com/78001660/185016399-416849f4-5e1c-4ed4-a71a-b6db20146293.png)

**Comment by Aman9das** [comment]
In file manager check whats inside temp_folder, there should be 2 `.img` files

**Comment by Midnightmadwalk-coder** [comment]
> In file manager check whats inside temp_folder, there should be 2 `.img` files

So you're saying that I should only paste those two files and remove the extra folder extracted as lineage os?
![Screenshot from 2022-08-17 07-40-49](https://user-images.githubusercontent.com/78001660/185019869-db27c11f-51e4-4a67-9b67-e209e29ef7eb.png)
![Screenshot from 2022-08-17 07-40-43](https://user-images.githubusercontent.com/78001660/185019880-a228ad73-a8a7-490c-abdf-d7d56757b086.png)

**Comment by Aman9das** [comment]
yes..
